test/acceptance/mixins/logger.acceptance.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, inject} from '@loopback/core';
+import {RestComponent, RestServer, get, param} from '@loopback/rest';
+import {sinon, Client, createClientForHandler} from '@loopback/testlab';
+import {LoggerMixin, Logger, LogArgs} from '../../..';
+
+describe('logger.mixin (acceptance)', () => {
+  // tslint:disable-next-line:no-any
+  let app: any;
+  let server: RestServer;
+  // tslint:disable-next-line:no-any
+  let spy: any;
+
+  beforeEach(createApp);
+  beforeEach(createLogger);
+  beforeEach(createController);
+  beforeEach(getServerFromApp);
+  beforeEach(() => {
```

Comment:
We try to use named hooks, see http://loopback.io/doc/en/contrib/style-guide.html#hooks